src/lib/geometry.f90

```diff
@@ -1309,7 +1309,7 @@ subroutine make_data_grid(surface_elems,spacing,to_export,filename,groupname)
       if(to_export)then
          !!! export vertices as nodes
          writefile = trim(filename)//'.exnode'
-         open(10, file = writefile, status='replace')
+         open(10, file = writefile)
```

Comment:
Please retain the line from my branch:
open(10, file = writefile, status='replace')